src/utils/routes.js

```diff
@@ -2,5 +2,7 @@ const ROUTES = {
   index: '/',
   login: '/login',
   register: '/register',
+  profiles: '/profile/:username',
```

Comment:
Hey @bevkololi ,
Maybe for redability you could have these routes as:
```
profiles: {
        create: '/profile/new/:username/'
        update:'/profile/edit/:username/'
}

```
Then you can reference them as `ROUTES.profiles.create` or `ROUTES.profiles.update`